### DISCUSSION TITLE
Command to read a txt

### DISCUSSION THREAD

**Discussion by Diego-Schmidt** [vasani-arpit/WBOT#220] [Q&A]
Hello, I'm totally noob in this js world... is there any way to use to bot to read and show the contents of a txt with a command? for example /calendar . How can i set the "admins" of the bot so that command can only be used by specific users?

**Comment by vasani-arpit** [comment]
Static and fixed responses can be set from `bot.json` file you can write your commands in `exact` property in `bot.json` file. Bot will reply accordingly.

**Reply by Diego-Schmidt** [reply]
Thanks for the reply and for your work, the bot is really easy to use. What do you mean whit "you can write your commands in exact property"? I'm just a basic user, i don't know anything about js :D. I promise I will learn and stop asking dumb questions :D

**Reply by Diego-Schmidt** [reply]
I was trying to understand how the system works experimenting with the webhook, using some code like this

`var fs  = require("fs");

if($message == '/calendar')
{
    var fs = require('fs');
	$response = fs.readFile('calendar.txt', 'utf8', function(err, contents) {
	// code using file data
	});
    $response = console.log(data);
    echo($response);
}`

But i don't think this make any sense, since i did not produce anything :D

**Comment by Diego-Schmidt** [comment]
I will reply myself, i repeat I'm learning, my knowledge of this is zero :P
I have added code to the php webhook to achieve part of what I'm looking for

`$agenda = file_get_contents("agenda.txt");
if($message == '/agenda')
{
	$response = array(array('text' => $agenda, 'type' => 'message'));
    $response = json_encode($response);
    echo($response);
}`

And that gives me the contents of the agenda.txt file... good, that is what i wanted. Now I need to learn how to store text on the agenda.txt file using a command. How can I pass the text after a command ? like /addtoagenda blah blah blah
